sklearn/datasets/_base.py

```diff
@@ -757,7 +757,9 @@ def load_digits(*, n_class=10, return_X_y=False, as_frame=False):
 
 @_deprecate_positional_args
 def load_diabetes(*, return_X_y=False, as_frame=False):
-    """Load and return the diabetes dataset (regression).
+    """Load and return the diabetes dataset (regression). The meaning of each
```

Comment:
Let's put the note after the table below using sphinx:

```
.. note::
   The meaning of each feature (i.e. `feature_names`) might be unclear (especially for `ltg`)
   as the documentation of the original dataset is not explicit. We provide information that
   seems correct in regard with the scientific literature in this field of research.
```

Reply:
Thanks for showing how sphinx can be used! Updated.